provisioning/common/etcd/values.yaml

```diff
@@ -56,6 +56,9 @@ extraEnvVars:
   # Default value in etcd v3.2+ is "100 000": https://etcd.io/docs/v3.5/op-guide/maintenance/#raft-log-retention
   - name: ETCD_SNAPSHOT_COUNT
     value: "10000"
+  # GOMEMLIMIT to prevent OOM kill, update the value together with "resources" spec.
+  - name: GOMEMLIMIT
+    value: "256MiB"
 
```

Comment:
Added `GOMEMLIMIT` to let the `etcd` know about memory limit.
This is default value, value is adjusted in `common.sh`, see `helm upgrade ...`.